platform-cloud/docs/tutorials/retry-strategy.md

```diff
@@ -0,0 +1,81 @@
+
+This guide outlines best practices for mitigating the impact of Spot interruptions and ensuring critical tasks can retry or recover reliably.
+
+## Recommended mitigations
+
+### Use an On-Demand compute environment
+
+For workflows with a significant proportion of long-running processes, the costs of, and mitigations necessary for working with Spot may outweigh the benefits. You may find it simpler and cheaper to run those workloads in On-Demand compute environments.
+
+### Move long-running tasks to On-Demand 
+
+Tasks with long runtimes are particularly vulnerable to Spot termination. If you don’t already have one, first create an On-Demand compute environment in Seqera Platform. Then, in Platform, you can explicitly assign critical or long-duration tasks to On-Demand queues and leave other tasks to run in a Spot queue by default: 
+
+```
+process {
+	withName: 'run_bcl2fastq' {
+	     queue = 'TowerForge-MyOnDemandQueue'
+	} 
+}
+```
+
+To find the queue name, open the **Compute Environments** tab in Seqera Platform, and open the relevant compute environment. Scroll down to the **Manual Config Attributes** section to view key configuration details, including queue names. Look for the queue name prefixed with `TowerForge-` if it was created by Forge. 
+
+### Use retry strategies for Spot Interruptions
+
+#### Handle retries in Nextflow by setting `errorStrategy` and `maxRetries`
+
+A retry strategy at the Nextflow level is more appropriate when run times are low and retries are likely to succeed. This can be configured as follows:
+
+```bash
+process {
+   errorStrategy = 'retry'
+   maxRetries = 3 
+}
+```
+
+This retry strategy in the above example configuration will all types of job failures. Currently, it is not possible to configure retries at the Nextflow level specifically for reclamations because Spot reclamations do not produce diagnostic exit codes. Note that, given the escalating costs of repeated retries, an On-Demand queue is likely a more cost-effective option than very large numbers of retries. If you still see failures after applying a configuration like this, solutions involving On-Demand queues are likely to be more effective at limiting costs and runtimes.
+
+#### Handle retries in AWS by setting `aws.batch.maxSpotAttempts`
+
+If you do not have processes with long runtimes in your workflow (i.e., all processes are under one hour runtime), you can consider configuring automatic retries in case of interruption:
```

Comment:
```suggestion
If all processes in your workflow have runtimes short enough to feasibly complete before reclamation, you can consider configuring automatic retries in case of interruption:
```

Reply:
Just so as not to be too precise with the run time thing